src/dom/tracked-listener.ts

```diff
@@ -0,0 +1,74 @@
+type EventTarget = HTMLElement|Window;
+type Listener = [EventTarget, string, EventListenerOrEventListenerObject];
+
+/**
+ * Used for tracking listeners by ID.
+ * This way event listeners in a component can be tossed into a set and easily
+ * removed during dispose. Otherwise which element had which events tied to
+ * which functions needs to be tracked in the class which can be annoying with
+ * dynamically added listeners.
+ */
+export class TrackedListener {
```

Comment:
Seems like there is a overlap with DomWatcher class.  Is DomWatcher something you can use instead?

Reply:
Done.